apps/damap-frontend/src/themes/custom-palettes.scss

```diff
@@ -37,7 +37,7 @@ $_palettes: (
     90: #d4e4f6,
     95: #e7f2ff,
     98: #f7f9ff,
-    99: #fcfcff,
+    99: #f6f6f6,
```

Comment:
The values were ascending before the PR. The change of the key 99 is introduced here

Reply:
David is correct that these values should be ascending - the values were automatically created via a m3 color tool (i gave it out primary color as input). Adding new color keys to the palette is fine, but i would suggest to always keep a logical ordering and to make the jumps "natural". In this case i dont see the need to change the color palette, since the colors are so similar anyways. I would also revert the 0 -> 00 change, since they were naturally generated this way and the other palettes still use 0 or 4 as keys.